web/src/main/java/com/stezhka/web/controller/MovieController.java

```diff
@@ -0,0 +1,47 @@
+package com.stezhka.web.controller;
+
+import com.stezhka.entity.Movie;
+import com.stezhka.service.MovieService;
+import com.stezhka.web.dto.MovieDto;
+import com.stezhka.web.util.JsonJacksonConverter;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping(value = "/movie", method = RequestMethod.GET, produces = "application/json;charset=utf-8")
```

Comment:
Please remove RequestMethod for class annotation and add it to each separate method

Reply:
Done